app/src/main/java/com/example/tour_a_versity_v3/TucInfo.kt

```diff
@@ -4,17 +4,15 @@ import android.content.Intent
 import android.os.Bundle
 import android.widget.Button
 import androidx.appcompat.app.AppCompatActivity
-import com.example.tour_a_versity_v3.CECH_Photos
-import com.example.tour_a_versity_v3.R
 
-class CECH_Info : AppCompatActivity() {
+class TucInfo : AppCompatActivity() {
```

Comment:
Changed to class name according to naming rules in Kotlin as explained here:
https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules